app/views/users/registrations/edit.html.erb

```diff
@@ -1,11 +1,11 @@
 <h2>Edit <%= resource_name.to_s.humanize %></h2>
 
 <%= form_with(model: resource, url: registration_path(resource_name), method: :put, data: { turbo: false }) do |f| %>
-  <%= render "users/shared/error_messages", resource: resource %>
```

Comment:
do we need to delete `"users/shared/error_messages"` or is it used somewhere else?

Reply:
I moved it to `views/shared` since it could be used for every model (not only for users).